libs/core/src/lib/text/text.component.scss

```diff
@@ -0,0 +1,9 @@
+@import '~fundamental-styles/dist/text';
+
+$block: fd-text;
+
+.#{$block} {
+    &__lineclamp {
```

Comment:
@EvilAlexei maybe it should be moved to fundamental-styles/text?

Reply:
+1, but you can keep it here till the next release, just add a comment that it should be removed after fund-styles is released

Reply:
Hi @InnaAtanasova, @Betrozov, thank you for your comments  :)

This style is needed for this specific implementation with the line clamp directive,
but not necessary for the text component in general,
I'm not sure that It will be needed, for example, for the same react component.